spec/truffle/launcher_spec.rb

```diff
@@ -31,11 +31,15 @@
   end
 
   it "prints the full java command with -J-cmd" do
-    out = `#{RbConfig.ruby} -J-cmd --version`
-    parts = out.split(' ')
-    parts[0].should == "$"
-    parts[1].end_with?("java").should == true
-    $?.success?.should == true
+    commands = ["#{RbConfig.ruby} -J-cmd --version",
+                "JAVA_OPTS='-cmd' #{RbConfig.ruby} --version"]
+    commands.each do |cmd|
+      out   = `#{cmd}`
+      parts = out.split(' ')
+      parts[0].should == "$"
+      parts[1].end_with?("java").should == true
+      $?.success?.should == true
+    end
```

Comment:
@graalvm/truffleruby One general note about specs: I don't mind a bit of duplication in specs.
It's usually easier to debug with some duplication than a loop, e.g. we don't know which command failed if one of these two fails, but we would if the expectations were duplicated.